charm/docs/tutorial/getting-started.md

```diff
@@ -1,31 +1,34 @@
+<!-- This tutorial should be updated in the src-docs directory. -->
+
 # Getting Started
 
 ## What you’ll do
+
 - Deploy the NetBox charm.
 - Integrate with Redis using the redis-k8s charm.
 - Integrate with the PostgreSQL K8s charm.
 - Integrate with S3 for storage.
 - Expose the NetBox charm with Traefik k8s.
 - Create a super user.
 
-Through the process, you'll verify the workload state, and log in to
+Through the process, you’ll verify the workload state, and log in to
 your NetBox instance.
 
 ## Requirements
+
 - Juju 3 installed.
 - Juju controller that can create a model of type kubernetes.
 - Read/write access to a S3 compatible server with a bucket created.
 - Configuration compatible with the traefik-k8s charms. In the case of MicroK8S this can be achieved with the metallb addon.
 
 For more information about how to install Juju, see [Get started with Juju](https://juju.is/docs/olm/get-started-with-juju).
```

Comment:
If the requirements items about Juju and bootstrapping are updated, then I think this line can be removed.

Reply:
That line was automatically added by sphinx.

Reply:
Ah okay, we can leave it alone then.